eox_core/settings/production.py

```diff
@@ -1,13 +1,18 @@
 """
 Settings for eox_core project meant to be called on the edx-platform/*/envs/aws.py module
 """
+import logging
+
 from .common import *  # pylint: disable=wildcard-import, unused-wildcard-import
 
+LOG = logging.getLogger(__name__)
+
 try:
     import sentry_sdk
     from sentry_sdk.integrations.django import DjangoIntegration
 except ImportError:
     sentry_sdk = DjangoIntegration = None
+    LOG.error("ImportError while importing %s", ImportError)
```

Comment:
what does this log? don't you need to include the exception instead of `ImportError`?

Reply:
Hi @mariajgrimaldi  sorry, yes I think if you need to include the exception. But it depends on the answer if you are going to do the same for sentry as what was done in PR [#254](https://github.com/eduNEXT/eox-core/pull/254), then it would no longer be necessary I think.